app/models/agents/user_location_agent.rb

```diff
@@ -14,6 +14,8 @@ class UserLocationAgent < Agent
       If you want to only keep more precise locations, set `max_accuracy` to the upper bound, in meters. The default name for this field is `accuracy`, but you can change this by setting a value for `accuracy_field`.
 
       If you want to require a certain distance traveled, set `min_distance` to the minimum distance, in meters. Note that GPS readings and the measurement itself aren't exact, so don't rely on this for precision filtering.
+
+      To view the locations on a map, set `api_key` to your Google Maps JavaScript API key.
```

Comment:
Maybe [link](https://developers.google.com/maps/documentation/javascript/get-api-key#key) this to https://developers.google.com/maps/documentation/javascript/get-api-key#key ?